databuilder/models/table_metadata.py

```diff
@@ -15,6 +15,7 @@ class TagMetadata:
     TAG_NODE_LABEL = 'Tag'
     TAG_KEY_FORMAT = '{tag}'
     TAG_TYPE = 'tag_type'
+    BADGE_TYPE = '^b'
```

Comment:
currently badge is special type of tag with attribute tag_type as badge. I prefer not to have "^b" to identify badge.

Reply:
this was just a way of supporting serialized string representation of it which I will either move to csv extractor or get rid of completely